101-automation-configuration/nested/provisionServer.json

```diff
@@ -0,0 +1,170 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "location": {
+      "type": "string"
+    },
+    "vmName": {
+      "type": "string"
+    },
+    "vmSize": {
+      "type": "string"
+    },
+    "adminUsername": {
+      "type": "string"
+    },
+    "adminPassword": {
+      "type": "securestring"
+    },
+    "storageAccountNamePrefix": {
+      "type": "string"
+    },
+    "dataDisk": {
+      "type": "string"
+    },
+    "nicName": {
+      "type": "string"
+    },
+    "automationAccountName": {
+      "type": "string"
+    }
+  },
+  "variables": {
+    "extAutomationMetaMOFURL": "https://eus2oaasibizamarketprod1.blob.core.windows.net/automationdscpreview/RegistrationMetaConfigV2.zip",
```

Comment:
is this a fixed file/location or will it change at some point and break deployment?  (e.g. preview)

Reply:
removed